sdk/synapse/Azure.Analytics.Synapse.Spark/samples/Sample1_SubmitSparkJobAsync.md

```diff
@@ -0,0 +1,68 @@
+
+## Create Spark batch client
+
+To interact with Spark jobs running on Azure Synapse, you need to instantiate a `SparkBatchClient`. It requires an endpoint URL and a `TokenCredential`.
+
+```C# Snippet:CreateSparkBatchClientAsync
+SparkBatchClient client = new SparkBatchClient(new Uri(endpoint), sparkPoolName, new DefaultAzureCredential());
+```
+
+## Submitting Spark jobs
+
+To submit a Spark job, first create a `SparkBatchJob`, passing in an instance of `SparkBatchJobOptions` describing the job's parameters. Calling `StartCreateSparkBatchJobAsync` with that job will submit it to Synapse.
+
+```C# Snippet:SubmitSparkBatchJobAsync
+string name = $"batch-{Guid.NewGuid()}";
+string file = string.Format("abfss://{0}@{1}.dfs.core.windows.net/samples/net/wordcount/wordcount.zip", fileSystem, storageAccount);
+SparkBatchJobOptions request = new SparkBatchJobOptions(name, file)
+{
+    ClassName = "WordCount",
+    Arguments =
+    {
+        string.Format("abfss://{0}@{1}.dfs.core.windows.net/samples/net/wordcount/shakespeare.txt", fileSystem, storageAccount),
+        string.Format("abfss://{0}@{1}.dfs.core.windows.net/samples/net/wordcount/result/", fileSystem, storageAccount),
+    },
+    DriverMemory = "28g",
+    DriverCores = 4,
+    ExecutorMemory = "28g",
+    ExecutorCores = 4,
+    ExecutorCount = 2
+};
+
+SparkBatchOperation createOperation = await client.StartCreateSparkBatchJobAsync(request);
+SparkBatchJob jobCreated = await createOperation.WaitForCompletionAsync();
+```
+
+## Retrieve a Spark job
+
+To retrieve the details of a Spark job call `StartGetSparkBatchJobAsync`, passing in the Spark job ID.
+
+```C# Snippet:GetSparkBatchJobAsync
+SparkBatchOperation getOperation = await client.StartGetSparkBatchJobAsync (jobCreated.Id);
+SparkBatchJob retrievedJob = await getOperation.WaitForCompletionAsync();
+
+Debug.WriteLine($"Job is returned with name {retrievedJob.Name} and state {retrievedJob.State}");
+```
+
+## List Spark jobs
+
+To enumerate all Spark jobs in the Synapse workspace call `GetSparkBatchJobs`.
+
+```C# Snippet:ListSparkBatchJobsAsync
+Response<SparkBatchJobCollection> jobs = client.GetSparkBatchJobs();
+foreach (SparkBatchJob job in jobs.Value.Sessions)
+{
+    Console.WriteLine(job.Name);
+}
+```
+
+## Canceling a Spark job
+
+To cancel a submitted Spark job call `CancelSparkBatchJob`, passing in the Spark job ID.
+
+```C# Snippet:DeleteSparkBatchJobAsync
+Response operation = client.CancelSparkBatchJob(jobCreated.Id);
```

Comment:
nit: snippet is Delete, but method name is Cancel.

Reply:
Done.